packages/reassure-compare/src/types.ts

```diff
@@ -57,6 +74,10 @@ export interface RemovedEntry {
   baseline: PerformanceEntry;
 }
 
+export interface Metadata {
```

Comment:
Since this type is for compare output lets add `Compare` to the name
```suggestion
export interface CompareMetadata {
```